tests/system/python/packages/test_statistics_history_notification_rule.py

```diff
@@ -0,0 +1,293 @@
+SCRIPTS_DIR_ROOT = "{}/tests/system/python/packages/data/".format(PROJECT_ROOT)
+FLEDGE_ROOT = os.environ.get('FLEDGE_ROOT')
+SOUTH_SERVICE_NAME = "Sine #1"
+SOUTH_ASSET_NAME = "{}_sinusoid_assets".format(time.strftime("%Y%m%d"))
+NOTIF_SERVICE_NAME = "notification"
+NOTIF_INSTANCE_NAME = "notify #1"
+AF_HIERARCHY_LEVEL = "{0}_teststatslvl1/{0}_teststatslvl2/{0}_teststatslvl3".format(time.strftime("%Y%m%d"))
+
+@pytest.fixture
+def reset_fledge(wait_time):
+    try:
+        subprocess.run(["cd {}/tests/system/python/scripts/package && ./reset"
+                       .format(PROJECT_ROOT)], shell=True, check=True)
+    except subprocess.CalledProcessError:
+        assert False, "reset package script failed!"
+
+    # Wait for fledge server to start
+    time.sleep(wait_time)
+
+
+@pytest.fixture
+def start_south(add_south, fledge_url):
+    south_plugin = "sinusoid"
+    config = {"assetName": {"value": SOUTH_ASSET_NAME}}
+    # south_branch does not matter as these are archives.fledge-iot.org version install
+    add_south(south_plugin, None, fledge_url, service_name=SOUTH_SERVICE_NAME, installation_type='package', config=config)
+
+
+@pytest.fixture
+def start_north(start_north_omf_as_a_service, fledge_url,
+                      pi_host, pi_port, pi_admin, pi_passwd, clear_pi_system_through_pi_web_api, pi_db):
+    
+    af_hierarchy_level_list = AF_HIERARCHY_LEVEL.split("/")
+    dp_list = ['sinusoid']
+    asset_dict = {}
+    asset_dict[SOUTH_ASSET_NAME] = dp_list
+    clear_pi_system_through_pi_web_api(pi_host, pi_admin, pi_passwd, pi_db,
+                                       af_hierarchy_level_list, asset_dict)
+
+    response = start_north_omf_as_a_service(fledge_url, pi_host, pi_port, pi_user=pi_admin, pi_pwd=pi_passwd,
+                                            default_af_location=AF_HIERARCHY_LEVEL)
+
+    yield start_north
+
+@pytest.fixture
+def add_notification_service(service_branch, fledge_url, wait_time):
+    try:
+        subprocess.run(["sudo apt install -y fledge-service-notification fledge-notify-asset"], shell=True, check=True)
+    except subprocess.CalledProcessError:
+        assert False, "notification service installation failed"
+
+    # Enable service
+    data = {"name": NOTIF_SERVICE_NAME, "type": "notification", "enabled": "true"}
+    print (data)
+    utils.post_request(fledge_url, "/fledge/service", data)
+
+    # Wait and verify service created or not
+    time.sleep(wait_time)
+    verify_service_added(fledge_url, NOTIF_SERVICE_NAME)
+
+    payload = {
+        "name": "test #1",
+        "description": "test notification instance",
+        "rule": "Threshold",
+        "rule_config": {
+            "source": "Statistics History",
+            "asset": "READINGS",
+            "trigger_value": "10.0",
+        },
+        "channel": "asset",
+        "delivery_config": {"enable": "true"},
+        "notification_type": "retriggered",
+        "retrigger_time": "30",
+        "enabled": True,
```

Comment:
Here we are using bool? I think in fledge we handle every type as string instead of their actual type.